GOVERNANCE.md

```diff
@@ -0,0 +1,477 @@
+    - If an exclusive choice has to be made (where it is not possible to entertain two conflicting approaches), and no clear side is more advantageous to pick, a vote should be held with the majority opinion being the final decision.
+
+- Some decisions are local, e.g. a plugin does not impact others with its behaviour, however it still needs to be a 'good citizen' in the project (CI, format, etc..)
+    - These should be clearly scoped in their respective project part
+    - However, if needed a 'higher up' decision can be requested if no consensus is achieved
+
+
+
+## Project Structure
+
+The project has a hierarchy to solve these two problems:
+
+1. How can users make sure that their voice and interests are represented in the project?
+    - Each member party appoints one core team member
+    - They form the group of people that have to, and are responsible for, signing off on the technical implementation of the project
+2. How to make the lives of these team members easier?
+
+
+-----
+
+* Thin-edge is organized around sub-projects - to ease decision taking.
+    * The sub-projects are organized in a hierarchy
+        * This makes it easier to decouple different parts of the project
+        * This makes it easier to decide on technical questions
+    * Everyone has to agree on the core project as it is the foundations to all others.
+    * At the periphery, sub-projects might be related to specific eco-systems (e.g. Cumulocity or Moneo) and therefore have independent decision processes.
+    *	At an intermediate level, one might have sub-projects related to specific use-cases like “JSON over MQTT”.
+* All the sub-projects share a common repository - to ease consistency across projects and over time.
+    *	Labels are used to organize issues/PRs/discussions along sub-projects.
+    *	Code ownership is used to enforce cooperation around key components, but the default is to let things open and to trust each other, using version control as a safety net.
+
+
+
+
+
+
+```
+core contributors = [SAG, IFM]
+
+-> Project leads, one each from [core contributors]
+|
+ \- Responsible for `/*`, aka everything in the repository
+ |
+ |
+ \
+  \-> Delegated project Teams, any combination of [teams, individual contributors]
+  |
+  |\- Responsible for `/plugins/plugin_{foo,bar}`
+  ||- Are trusted by the project leads (who would have to stand-in for the trusted members)
+  | 
+  |->  Can be needed to nest further if the project grows bigger
+```
+
+## Repository maintenance
+
+To assure a consistent level of quality and appearance, the project has some rules for maintainers to follow:
+
+- **Clear commits**
+  Having clear, concise and well-written commit messages for all changes is not
+  only a indicator for the seriousness of the project, but also a way to ensure
+  sustainable development via repeatability of the sources of the project.
```

Comment:
```suggestion
  sustainable development via replicability of the sources of the project. 
```

This was broken in ea79548aaf16e2f23e272cd51200e19050a794d9, "replicability" is the right word here.

Reply:
Applied - even if not used to that term.